### ISSUE TITLE
Problem with YAML config

### ISSUE THREAD

**Issue by aserrallerios** [devopsfaith/krakend#198]
Hello,

it looks like KrakenD is ignoring my JOSE block in YAML (not in JSON). This looks like a valid YAML, but the request returns 200 with no token (it should return 401).

```yaml
version: 2
timeout: 3000ms
cache_ttl: 300s
host:
- "http://fake_api:8080"
extra_config:
  github_com/devopsfaith/krakend-gologging:
    level: DEBUG
    prefix: "[KRAKEND]"
    syslog: false
    stdout: true
endpoints:
- endpoint: "/{p1}"
  headers_to_pass:
  - Authorization
  backend:
  - url_pattern: "/{p1}"
    sd: static
    encoding: json
  extra_config:
    github.com/devopsfaith/krakend-jose/validator:
      alg: RS256
      jwk-url: "https://a.b.com/oauth2/default/v1/keys?client_id=xxx"
      cache: true
  concurrent_calls: 1
output_encoding: json
```

I'm actually not sure if I should post this in krakend-ce. Please close if this is not the right place.

**Comment by kpacha** [comment]
hi, @aserrallerios 

you are right, the parsed configuration has some malformed `extra_config` structs the system does not understand.

thks for the heads-up

**Comment by kpacha** [comment]
it should be fixed at master, and will be included into the next release (0.8.0)
